doc/user/content/connect-sources/postgres-amazon-rds.md

```diff
@@ -0,0 +1,395 @@
+
+When your PostgreSQL database is running in a private network, there are various ways to allow Materialize to connect:
+
+- **Allow Materialize IPs:** This is the simplest approach. You configure your database's security group to allow public connections from a set of static Materialize IP addresses and rely on PostgreSQL authentication and authorization to keep your data safe.
+
+- **Use AWS PrivateLink** or **Use an SSH tunnel:** If you need an additional layer of security at the network level, you can use either [AWS PrivateLink](https://aws.amazon.com/privatelink/) or an SSH tunnel to give Materialize secure access to your database.
+
+Select the option that works best for you.
+
+{{< tabs >}}
+
+{{< tab "Allow Materialize IPs">}}
+
+1. Connect `psql` to Materialize.
+
+    You can find your `psql` connection command on the **Connect** screen in the [Materialize UI](https://console.materialize.com/).
+
+1. Find your static egress IP addresses:
+
+    ```sql
+    SELECT * FROM mz_egress_ips;
+    ```
+1. In the RDS Console, [add an inbound rule](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/working-with-security-groups.html#adding-security-group-rule) to your database's security group:
+
+    Type | Protocol | Port range | Source
+    -----|----------|------------|-------
+    Custom TCP | TCP | `5432` | IP address from the previous step
+
+{{< /tab >}}
+
+{{< tab "Use PrivateLink">}}
+{{< /tab >}}
+
+{{< tab "Use an SSH tunnel">}}
+{{< /tab >}}
+
+{{< /tabs >}}
+
+## Step 4. Create an ingestion cluster
+
+In Materialize, a [cluster](/get-started/key-concepts/#clusters) is an **isolated environment**, similar to a virtual warehouse in Snowflake. Within a cluster, you have [replicas](/get-started/key-concepts/#cluster-replicas), which are the **physical resources** for doing computational work. Clusters are completely isolated from each other, so replicas can be sized based on the specific task of the cluster, whether that is ingesting data from a source, computing always-up-to-date query results, serving results to clients, or a combination.
+
+In this case, you'll create 1 new cluster containing 1 replica for ingesting source data from your PostgreSQL database.
+
+1. Connect `psql` to Materialize.
+
+    You can find your `psql` connection command on the **Connect** screen in the [Materialize UI](https://console.materialize.com/).
+
+1. Use the [`CREATE CLUSTER`](/sql/create-cluster/) command to create the new cluster and replica:
+
+    ```sql
+    CREATE CLUSTER ingest_rds_postgres REPLICAS (r1 (SIZE = 'medium'));
+    ```
+
+    At first, it's recommended to use a `medium` [size](/sql/create-cluster-replica/#sizes) replica or larger. This will help Materialize more quickly take an initial snapshot of the tables in your publication. Once the snapshot is finished, you'll right-size the cluster replica.
+
+1. Use the [`SHOW CLUSTER REPLICAS`](/sql/show-cluster-replicas/) command to check the status of the replica:
```

Comment:
Should we skip this step? You can create the source before the replica is ready, and Materialize is smart enough to wait for the replica to come online before attempting to ingest data.